activemq-client/src/main/java/org/apache/activemq/ActiveMQProducer.java

```diff
@@ -87,7 +88,7 @@ public JMSProducer send(Destination destination, Message message) {
                 }
             }
 
-            activemqMessageProducer.send(destination, message, getDeliveryMode(), getPriority(), getTimeToLive(), getDisableMessageID(), getDisableMessageTimestamp(), null);
+            activemqMessageProducer.send(destination, message, getDeliveryMode(), getPriority(), getTimeToLive(), completionListener);
```

Comment:
`getDisableMessageID()` and  `getDisableMessageTimestamp()` have been removed as parameters to `send`.
I guess there's no scenario where `disableMessageID` and `disableMessageTimestamp` could be set on `ActiveMQProducer` but not on `ActiveMQMessageProducer`?

Reply:
Those are passed internally in that method. Let me double check.

Reply:
Actually, I changed the logic here.